desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -215,6 +220,7 @@ def handle_adls_req_mapping(self, method, params, relative_path):
         relative_path += lib_urlunquote(params['directory'])
     
     if method == 'PUT':
+      access_type = 'rename-source'
```

Comment:
cf. past review this feels fishy, add headers?

Reply:
When sending the old dir to get its token, it wont be having that header. Will try and test if its a problem for fetching its SAS token @romainr.

Reply:
Rename has 2 calls to raz client, one without header (to fetch the token for header path) and the other having header (header path signed with previous fetched token). 

So not able to distinguish in the mapping to allow both of them for this acess_type but at the same time, check for header in both as former doesn't have it but needs this access_type.